requirements.txt

```diff
@@ -1,2 +1,3 @@
+mock
```

Comment:
Why is this needed in `requirements.txt` if it's already part of `test-requirements.txt`?

Reply:
Pycharm added it. Fixed it.